grails-app/controllers/org/pih/warehouse/api/ApiController.groovy

```diff
@@ -155,7 +155,8 @@ class ApiController {
         String minimumExpirationDate = dateFormat.format(grailsApplication.config.openboxes.expirationDate.minValue)
         def logoLabel = grailsApplication.config.openboxes.logo.label
         def pageSize = grailsApplication.config.openboxes.api.pagination.pageSize
-        def logoUrl = location?.logo ? "/openboxes/location/viewLogo/${session.warehouse?.id}" : null
+        def logoUrl = location?.logo ? "/openboxes/location/viewLogo/${session.warehouse?.id}" : grailsApplication.config.openboxes.logo.url
```

Comment:
Additionally please use `createLink` here and use `contextPath` in the `grailsApplication.config.openboxes.logo.url`